module-4/backend/week_one.md

```diff
@@ -1,21 +1,49 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+basic JS syntax/vocabulary - it helps me talk through problems, and set up simple code (that can be then be addressed/lends to my google-fu)
+
 2. What are some tools to help debug JavaScript code?
+dev tools: source code/debugger/console/element/network
+
 3. What are some tools you need in order to unit test your JavaScript?
+mocha/chai
+
 4. What is the syntax for invoking a function?
+functionName() - the parentheses at the end
+
 5. What is CORS?
+Cross Origin Resource Sharing
```

Comment:
Questions 5 & 6 are from an old version of the repo, please pull the latest so you will have the updated questions in the future